Sliding-Images-Easy.md

```diff
@@ -0,0 +1,247 @@
+# Sliding Images Easy
+
+This tutorial will walk you through building an image slider using the [jQuery](https://jquery.com/) library.
+
+This tutorial will have three parts:
+- [HTML](#html)
+- [SCSS](#scss)
+- [JS](#js)
+- [References](#references)
+
+## HTML
+
+We will be using [Bootstrap](http://getbootstrap.com/) for this tutorial to keep things looking good, without spending a lot of time.
+
+Our structure will be as follows:
+
+```html
+<div class="container">
+
+  <!-- The wrapper for our slider -->
+  <div id="slider">
+    <ul class="slides"><!-- Each image will be inside this unordered list --></ul>
+  </div>
+
+  <div class="buttons"><!-- Pause and play buttons will go in here --></div>
+
+</div>
+```
+
+Inside our `ul` with the class of `slides` we will have the following:
+
+> Note: We repeat the first image at the end because the animation slides from the left.
+
+```html
+<li class="slide"><img src="" /></li>
+<li class="slide"><img src="" /></li>
+<li class="slide"><img src="" /></li>
+<li class="slide"><img src="" /></li>
+<li class="slide"><img src="" /></li>
+<li class="slide"><img src="" /></li>
+```
+
+Inside our `.buttons` class you should have the following:
+
+```html
+<button type="button" class="btn btn-default pause">
+	<span class="glyphicon glyphicon-pause"></span>
+</button>
+<button type="button" class="btn btn-default play">
+	<span class="glyphicon glyphicon-play"></span>
+</button>
+```
+
+Here is an example of what your html should look like:
+
+> Note: You should replace the image `src` attribute with your own content.
+
+```html
+<div class="container">
+
+  <div id="slider">
```

Comment:
use a class instead of and ID `"slider"` as this can be extended to multiple sliders in the page.